### ISSUE TITLE
Authentication Expired, now cannot re-authenticate

### ISSUE THREAD

**Issue by invertedworld** [mattrayner/pod-point-home-assistant-component#68]
I received an ‘Authentication Expired’ message. When I try to re-authenticate I get ‘ Email/Password authentication failed.’

I have changes and verified the password works in the PodPoint app. 

Logger: podpointclient.helpers
Source: custom_components/pod_point/config_flow.py:128
integration: Pod Point ([documentation](https://github.com/mattrayner/pod-point-home-assistant-component), [issues](https://github.com/mattrayner/pod-point-home-assistant-component/issues))
First occurred: 08:46:38 (1 occurrences)
Last logged: 08:46:38

Error updating access token. Auth Error (200) - Error processing access token response. 'idToken' not found in json.

Not sure where in HA to find the config to share.

**Comment by mattrayner** [comment]
Hi, which version of the plugin and home assistant are you running? The
latest version needs a new home assistant version so you will only see it
once HA is updated to 2024.7.1 or newer

You should be running 2.0.5

**Comment by invertedworld** [comment]
Hi Matt. 

Core 2024.10.2
Supervisor 2024.10.0
Operating System 13.1
Frontend 20241002.3

Pod Point component version 2.0.5

**Comment by garybrowngeo** [comment]
Same problem here. I think its where PodPoint have added 2FA.

**Comment by mattrayner** [comment]
It’s entirely possible that this has killed the component, this week I’ll
try and reverse engineer how it’s done

**Comment by garybrowngeo** [comment]
I have managed to remove the 2FA from my account. To do this you need to add the 2FA on the App and then go in again and remove it. You will then get an email with a link to formally remove the 2FA. Once this was done my HA component was able to sign in again. Hope this helps!

**Comment by invertedworld** [comment]
This worked for me too. Thanks!

**Comment by rozza-m** [comment]
Just encountered the same thing :(

**Comment by adamlc** [comment]
Maybe signing up for 2 accounts is a solution? Not ideal I know!

**Comment by leviowen** [comment]
Hi I’m new here. Thank you to the creator of this integration! Unfortunately going through the same thing as everyone above. We use the podpoint public charging a lot so not an option for us to turn off 2FA just to be able to use the integration :(

**Comment by colindawson** [comment]
The need for 2FA when using public chargers is a huge blocker for me too.  Hopefully this is something that can be addressed.   Fingers crossed. (I get the feeling that this is not so straight forward though)

**Comment by mattcd216** [comment]
Hi all. Is 2FA still a blocker? I have just this week needed to reauthenticate the integration, but getting an authentication error. I can log in to the official PP app fine (so credentials are ok).

**Comment by keithellis74** [comment]
Yes, I think this is still an issue.To use the PodPoint app you need 2FA set. The integration does not work with 2FA, so you can use one or the other. Not both. That is assuming you can still turn off 2FA in the PodPoint account.Keith EllisOn 14 Nov 2025, at 09:01, mattcd216 ***@***.***> wrote:﻿mattcd216 left a comment (mattrayner/pod-point-home-assistant-component#68)
Hi all. Is 2FA still a blocker? I have just this week needed to reauthenticate the integration, but getting an authentication error. I can log in to the official PP app fine (so credentials are ok).
